Sources/Tracing/Docs.docc/Guides/InstrumentYourLibrary.md

```diff
@@ -183,83 +183,83 @@ protocol SampleHTTPHandler {
 }
 ```
 
-You may want to restore the baggage once around both those calls, or if that is not possible, restore it every time when calling back into user code, e.g. like this:
+You may want to restore the context once around both those calls, or if that is not possible, restore it every time when calling back into user code, e.g. like this:
 
 ```swift
 actor MySampleServer { 
   
-  var baggage: Baggage = .topLevel
+  var context: ServiceContext = .topLevel
   var userCode: SampleHTTPHandler
 
   func onHeaders(headers: HTTPHeaders) async {
-    await Baggage.withValue(self.baggage) { 
+    await ServiceContext.withValue(self.context) { 
       await userHandler.requestHeaders(headers)
```

Comment:
Should this be `userCode` instead of `userHandler`?

Reply:
I think `userHandler` is better, maybe rename the property declaration.

Reply:
handler is ok, thanks